exampleCourse/questions/templateRandomMultipleChoiceNumericChoices/question.html

```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+If $a = {{{params.a}}}$ and $b = {{{params.b}}}$, then what would the value of $a \times b$ be? 
```

Comment:
```suggestion
If $a = {{params.a}}$ and $b = {{params.b}}$, then what is $a$ multiplied by $b$? 
```

I think we should always double-brace for safety. And my eyes couldn't handle the use of a cross-product for scalar multiplication 😄